README.rst

```diff
@@ -17,23 +17,47 @@
 .. |black| image:: https://img.shields.io/badge/code%20style-black-000000.svg
 
 
-`skrub <https://skrub-data.github.io/>`_ (formerly *dirty_cat*) is a Python
+**skrub** (formerly *dirty_cat*) is a Python
 library that facilitates prepping your tables for machine learning.
 
 If you like the package, spread the word and ⭐ this repository!
 You can also join the `discord server <https://discord.gg/ABaPnm7fDC>`_.
 
-What can `skrub` do?
---------------------
-
-`skrub` provides data assembling tools (``TableVectorizer``, ``fuzzy_join``...) and
-encoders (``GapEncoder``, ``MinHashEncoder``...) for **morphological similarities**,
-for which we usually identify three common cases: **similarities, typos and variations**
+Website: https://skrub-data.org/
+
+What can skrub do?
+------------------
+
+The goal of skrub is to facilitate building and deploying machine-learning
```

Comment:
I don't think it helps with "deploying"

Reply:
Ah you're right, I took this from the VISION.rst

I'll just keep the building part